tasks/miarecweb.yml

```diff
@@ -197,6 +197,23 @@
     - reload celeryd
     - reload celerybeat
 
+- name: Configure Redis TLS in production.ini file
+  replace:
+    path: "{{ deploy_helper.new_release_path }}/production.ini"
+    regexp: "{{ item.regexp }}"
+    replace: "{{ item.replace }}"
+  with_items:
+    - regexp: 'redis://'
+      replace: 'rediss://'
+    - regexp: '%\(REDIS_PORT\)s/1$'
+      replace: '%(REDIS_PORT)s/1?ssl_cert_reqs=none'

```

Comment:
@timothylytle Redis is configured multiple times in `production.ini`. There are multiple components in MiaRec using redis for their own purpose and each of them have own connection URL:

```
celery.broker_url = redis://%(REDIS_HOST)s:%(REDIS_PORT)s/1
celery.result_backend = redis://%(REDIS_HOST)s:%(REDIS_PORT)s/1

dogpile_cache.arguments.url = redis://%(REDIS_HOST)s:%(REDIS_PORT)s/2?socket_timeout=%(REDIS_SOCKET_TIMEOUT)s&socket_connect_timeout=%(REDIS_SOCKET_CONNECT_TIMEOUT)s&socket_keepalive=%(REDIS_SOCKET_KEEPALIVE)s&health_check_interval=%(REDIS_HEALTH_CHECK_INTERVAL)s&pool_recycle=%(REDIS_POOL_RECYCLE)s

redis.sessions.url = redis://%(REDIS_HOST)s:%(REDIS_PORT)s/0?socket_timeout=%(REDIS_SOCKET_TIMEOUT)s&socket_connect_timeout=%(REDIS_SOCKET_CONNECT_TIMEOUT)s&socket_keepalive=%(REDIS_SOCKET_KEEPALIVE)s&health_check_interval=%(REDIS_HEALTH_CHECK_INTERVAL)s&pool_recycle=%(REDIS_POOL_RECYCLE)s

redis.cache.url = redis://%(REDIS_HOST)s:%(REDIS_PORT)s/3?socket_timeout=%(REDIS_SOCKET_TIMEOUT)s&socket_connect_timeout=%(REDIS_SOCKET_CONNECT_TIMEOUT)s&socket_keepalive=%(REDIS_SOCKET_KEEPALIVE)s&health_check_interval=%(REDIS_HEALTH_CHECK_INTERVAL)s&pool_recycle=%(REDIS_POOL_RECYCLE)s
```

Reply:
Understood, resolving this comment